x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldMapper.java

```diff
@@ -59,13 +57,6 @@ protected void parseCreateField(ParseContext context) {
         // there is no lucene field
     }
 
-    @Override
-    protected void doXContentBody(XContentBuilder builder, boolean includeDefaults, Params params) throws IOException {
-        super.doXContentBody(builder, includeDefaults, params);
-        AbstractScriptMappedFieldType fieldType = (AbstractScriptMappedFieldType) fieldType();
-        fieldType.mapperXContentBody(builder, params);
-    }
```

Comment:
see also #59968 . ParametrizedFieldMapper redefines toXContent hence doContentBody is never called. I proposed a change that makes it final so this method is not left behind when migrating an existing mapper.